<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputerInfoDialog</class>
 <widget class="QDialog" name="ComputerInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_computer_photo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Photo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_related_scientists">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Related scientists:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="info_related_scientists">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_comptuer_name">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_computer_Year_Built">
      <property name="text">
       <string>Year Built :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lable_Computer_Development">
      <property name="text">
       <string>Development :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_computer_Type">
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>10</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="computer_display_Name"/>
    </item>
    <item>
     <widget class="QLineEdit" name="computer_display_Year_Built"/>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QRadioButton" name="radioButton_display_if_developed">
       <property name="text">
        <string>Developed</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_display_if_original">
       <property name="text">
        <string>Original</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="computer_display_Type"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="info_computer_info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
